Lib/fontbakery/profiles/glyf.py

```diff
@@ -127,3 +127,36 @@ def com_google_fonts_check_glyf_non_transformed_duplicate_components(ttFont):
     else:
         yield PASS, ("Glyphs do not contain duplicate components which have"
                      " the same x,y coordinates.")
+
+@check(
+    id = 'com.google.fonts/check/glyf_nested_components',
+    rationale = """
+        There have been bugs rendering variable fonts with nested components.
+        Additionally, some static fonts with nested components have been
+        reported to have rendering and printing issues. (See
+        googlefonts/fontbakery#2961 and arrowtype/recursive#412.)
+    """,
+    conditions = ['is_ttf']
+)
```

Comment:
please add a misc_metadata field here with the link to the github issue where the check was originally proposed/requested. You can see how other checks declare it.